src/main/resources/assets/api-docs.yaml

```diff
@@ -2415,6 +2415,76 @@ paths:
             application/json:
               schema:
                 type: string
+  /api/user/acknowledgements/{key}:
```

Comment:
also: we tend to add these into their own files. so, e.g., this might go to `userAcknowledgementsByKey.yaml`, and the other endpoint into `userAcknowledgements.yaml`